### PR TITLE
match domain paths match case-insensitively

### PR DESCRIPTION
**Pull request by birarda**
https://highfidelity.manuscript.com/f/cases/edit/19565/Domain-paths-should-not-be-case-sensitive

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15314/HighFidelity-Beta-PR14450-b0aa1b2.apk

**Comment by antoninahifi** [comment]
build this please

**Comment by antoninahifi** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15758/HighFidelity-Beta-PR14450-b0aa1b2.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15758/HighFidelity-Beta-PR14450-b0aa1b2.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15758/HighFidelity-Beta-Interface-PR14450-b0aa1b2.exe

**Comment by antoninahifi** [comment]
currently testing

**Comment by antoninahifi** [comment]
PASS

File: domain-server/src/DomainServer.cpp [hunk 1]
```diff
@@ -3172,23 +3172,33 @@ void DomainServer::processPathQueryPacket(QSharedPointer<ReceivedMessage> messag
...
-            // if we didn't match the path BUT this is for the index path then send back our default
-            if (pathMatch.isValid()) {
-                responseViewpoint = pathMatch.toMap()[PATH_VIEWPOINT_KEY].toString();
-            } else {
-                const QString DEFAULT_INDEX_PATH = "/0,0,0/0,0,0,1";
-                responseViewpoint = DEFAULT_INDEX_PATH;
+            // enumerate the paths and look case-insensitively for a matching one
+            for (auto it = pathsMap.constKeyValueBegin(); it != pathsMap.constKeyValueEnd(); ++it) {
+                if ((*it).first.toLower() == lowerPathQuery) {
+                    responseViewpoint = (*it).second.toMap()[PATH_VIEWPOINT_KEY].toString().toLower();
```
Comment (Atlante45) [inline domain-server/src/DomainServer.cpp]: Should we be lowering the response?

Comment (birarda) [reply domain-server/src/DomainServer.cpp]: The viewpoint returned is position and orientation - no need

**Pull request merged**